src/site/layouts/press_releases_listing.drupal.liquid

```diff
@@ -0,0 +1,88 @@
+{% comment %}
+Example data:
+"entityUrl": {
+"breadcrumb": [
+{
+"url": {
+"path": "/",
+"routed": true
+},
+"text": "Home"
+}
+],
+"path": "/pittsburgh-health-care"
```

Comment:
Should we be hardcoding this?   Or is this mock data?

Reply:
Mock data. See line 2 `Example data:`

Reply:
@kevwalsh is correct, @swirtSJW